### DISCUSSION TITLE
Possible to (solar) charge below 6A?

### DISCUSSION THREAD

**Discussion by HA-TB303** [serkri/SmartEVSE-3#191] [General]
Hi,

Would it be possible to lower the minimum 6A charging current somehow?
Tesla supports 1-32A per fase and that would improve solar charging efficiancy a lot.

I am now lowering to 1-5A  via the tesla API, and that works, but is too slow and unreliable.

Thanks,

Michel

**Comment by dingo35** [comment]
I'm sorry but we're sticking to the standards here; feel free to compile your own version.

Please note that if you charge at those low currents, up to 1A might be wasted due to overhead of car electronics that are switched on. ..

**Comment by HA-TB303** [comment]
Understood.
In some (bad weather) cases this would be really beneficial, but you have a point for the 1-2A and overhead. I will limit that to 3 or 4A. 

![CleanShot 2023-08-03 at 11 50 38](https://github.com/serkri/SmartEVSE-3/assets/9609805/e11fb9f4-b442-478e-bd7c-0fea9e993e4f)


It would be nice if there would be an easy way to override (at your own risk off-course) as I am not able to compile myself.

Cheers,

Michel

**Reply by Ferforfaen** [reply]
I would like that option too, and i am not capable of compiling either :)

**Comment by Imaginous** [comment]
I agree it could be an expert feature.
It would be nice if you can edit the raw configuration file in the web GUI. Or set the min/max value via the API.
